ldurniat/berry.lua

```diff
@@ -15,7 +15,7 @@ local json = require 'json'
 --                                  MODULE                                                     --												
```

Comment:
General note: 

We need stick to one style (notation) e.g. 

`functionName(arq) vs functionName( arq )`

or

```
     end
end
```
vs
```
    end
(blank line)
end
```

I prefer second.
I think you know what I mean:)

Reply:
Definitely agree.  I initially kept the same style but as I kept adding more changes and moving blocks of code around things did indeed get a bit sloppy.

Reply:
So before I finish the changes to fix this I need to know how you want things done.

Specifically regarding method/variable naming style.

Right now variables and functions both use the same naming styles.  This is mostly camelCase but sometimes things don’t even have capitilzation.  (ie.  `hasbit` `setbit`)

I would really recommend to use camelCase for functions and methods.  Snake_case can be used for variables.  If you have a different preference let me know.

Reply:
I agree with you.

Reply:
Commit fd9f3fe fixes some of this.  Going to finish on knocking out the naming notation for variables and functions soon.

Reply:
I will check it later. Thanks.